src/main/java/com/itschool/eventmanagment/controllers/EventController.java

```diff
@@ -49,4 +52,17 @@ public ResponseEntity<List<EventDTO>> getFilteredAndSortedEvents(@RequestParam @
                                                                      @RequestParam(required = false) String sortBy) {
         return ResponseEntity.ok(eventService.getEvents(to, from, location, sortBy));
     }
+    @DeleteMapping("/{id}")
+    public ResponseEntity deleteEvent(@PathVariable("id") Long id){
+        eventService.deleteEvent(id);
+        return ResponseEntity.ok("Event delete succesfully");
+    }
+    @PutMapping("/{id}")
+    public ResponseEntity<EventDTO> updateEvent(@PathVariable Long id, @RequestBody EventDTO eventDTO){
+        if (id!= eventDTO.getId()){
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+        }
+       EventDTO updatedEvent = eventService.updateEvent(eventDTO);
+        return ResponseEntity.ok(updatedEvent);
```

Comment:
I don't advice having business logic on the controller layer. We can move this to the service layer.